lib/_http_server.js

```diff
@@ -683,6 +685,10 @@ function onSocketPause() {
   }
 }
 
+function socketSetEncoding() {
+  throw new ERR_HTTP_INCOMING_SOCKET_ENCODING('setEncoding');
```

Comment:
There is no dynamic argument in this error, so there is no need to pass through any arguments.